gnovm/pkg/gnolang/preprocess.go

```diff
@@ -5497,3 +5500,184 @@ func SaveBlockNodes(store Store, fn *FileNode) {
 		return n, TRANS_CONTINUE
 	})
 }
+
+// Check for invalid arguments in builtin functions
+// len, cap, make
+const (
+	BuiltinLen  = "len"
+	BuiltinCap  = "cap"
+	BuiltinMake = "make"
+)
```

Comment:
We don't use constant names, so just use the raw strings in the comparison above.